features/extra/world.js

```diff
@@ -1,11 +1,13 @@
 var world = require('./helpers');
+var fs = require('fs');
 var path = require('path');
 
-world.AWS = require('../../lib/aws');
 try {
-  world.AWS.config.loadFromPath(path.resolve('./configuration'));
+  world.config = JSON.parse(
+    fs.readFileSync(path.resolve('./configuration'))
```

Comment:
Can you copy `./configuration.sample` from V2 also? It will show an example of how user should config the region or the testing table name

Reply:
The file was copied in PR https://github.com/aws/aws-sdk-js-v3/pull/479
The file can be viewed at https://github.com/aws/aws-sdk-js-v3/blob/0ade538f67f817ae3b683b206ddfc6985bc7e6d7/configuration.sample

#479 was merged to keep the PR in this diff specific to dynamodb